### DISCUSSION TITLE
[Control Over Output] Type Setting on Grids

### DISCUSSION THREAD

**Discussion by sirinath** [typst/typst#194] [Ideas]
Is it possible to add the ability to ensure the lines of text in front of back of print aligns perfectly to reduce visual bleed through when printed on thin paper.

**Comment by laurmaedje** [comment]
It might be possible to build something like this with paragraph show rules and the `locate` function, but it's not supported natively at the moment.

**Reply by sirinath** [reply]
Please keep this in the pipeline and add it as an easy to use setting in the near future as this is important in printing certain texts.

**Reply by sirinath** [reply]
The wiki section on ConTeX Garden grid typing - https://wiki.contextgarden.net/Grid_typesetting
